### PR TITLE
74.0.0

### PR DESCRIPTION
**Pull request by romoore**
Internal testing at: cl/229543405

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -1,2 +1 @@
-# Do not add git-lfs attributes to this files. It causes problems with Cocoapods.
-# snapshot_test_goldens/**/*.png filter=lfs diff=lfs merge=lfs -text
+snapshot_test_goldens/**/*.png filter=lfs diff=lfs merge=lfs -text
```
Comment (romoore) [inline .gitattributes:1]: If https://github.com/material-components/material-components-ios/pull/6350 merges, then these changes will be ignored.

**Comment by romoore** [comment]
Had to revert the ChipTextField commits because they failed to compile internally. Looks like they're missing import statements.

**Comment by romoore** [comment]
CLA's already ok. All authors have already signed before merging to `develop`.

**Comment by andrewoverton** [comment]
Was #6350 supposed to fix the conflicts in `.gitattributes`?

**Comment by romoore** [comment]
It will - the new approach is a "custom strategy" that simply doesn't merge anything.

**Review by yarneo — APPROVED** [review]

**Pull request merged**